openlibrary/templates/site/alert.html

```diff
@@ -9,6 +9,7 @@
       <div class="language-component header-dropdown">
         <details>
           <summary>
+            <span>$(supported_langs[get_lang() or 'en']['native']) ($(get_lang() or 'en'))</span>
```

Comment:
This will error if the user language is not supported! Eg if `get_lang()` returns `sq`. Use the python `.get` to handle this case as well, eg 

```
$ ui_lang = get_supported_languages().get(get_lang()) or get_supported_languages()['en']
```

Reply:
Oh note this also means we'll have to change what's inside the brackets, otherwise for unsupported languages (eg Albanian which has language code `sq`) we'll see something like: "English (sq)" which is incorrect